src/Common/perf/PerfRunner/project.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.0.0-*",
```

Comment:
If we decide to keep this code in CoreFX I will change this over to new csproj tooling.